packages/elements/src/tree/helpers/types.ts

```diff
@@ -1,6 +1,6 @@
 import { CollectionComposer, DataItem } from '@refinitiv-ui/utils';
 
-export type TreeData<T extends TreeDataItem = TreeDataItem> = T[] | CollectionComposer<T> | null;
```

Comment:
Why did you remove the default?

Reply:
Both of `TreeDataItem` are the same thing, no need to add the default, Am I understand correctly?

Reply:
Is this an internal type only?

Reply:
Or will users use this type?

Reply:
Oh, I got it, this is the same default parameter concept. yes, it shouldn't be removed I'll fix it.